commit 53b448b609adffb4eb6b3471a5f1d0a50734fcc1 (refs/changes/95/295/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-08-16T10:05:16-07:00 (3 years, 2 months ago)
    
    joyent/node-verror#40 add contribution guidelines that reference cr.joyent.us

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
new file mode 100644
index 0000000..4192e25
--- /dev/null
+++ b/CONTRIBUTING.md
@@ -0,0 +1,19 @@
+# Contributing
+
+This repository uses [cr.joyent.us](https://cr.joyent.us) (Gerrit) for new
+changes.  Anyone can submit changes.  To get started, see the [cr.joyent.us user
+guide](https://github.com/joyent/joyent-gerrit/blob/master/docs/user/README.md).
+This repo does not use GitHub pull requests.
+
+See the [Joyent Engineering
+Guidelines](https://github.com/joyent/eng/blob/master/docs/index.md) for general
+best practices expected in this repository.
+
+Contributions should be "make prepush" clean.  To run "make check", you'll need
+these tools:
+
+* https://github.com/davepacheco/jsstyle
+* https://github.com/davepacheco/javascriptlint
+
+If you're changing something non-trivial or user-facing, you may want to submit
+an issue first.
diff --git a/README.md b/README.md
index b6d13ff..bd30b95 100644
--- a/README.md
+++ b/README.md
@@ -474,11 +474,4 @@ Returns an array of the errors used to construct this MultiError.
 
 # Contributing
 
-Contributions should be "make prepush" clean.  To run "make check", you'll need
-these tools:
-
-* https://github.com/davepacheco/jsstyle
-* https://github.com/davepacheco/javascriptlint
-
-If you're changing something non-trivial or user-facing, you may want to submit
-an issue first.
+See separate [contribution guidelines](CONTRIBUTING.md).
